src/components/capture/DetailsForm.tsx

```diff
@@ -66,12 +69,18 @@ function NewCaptureDetails({ connectorTags }: Props) {
                         connectorTags.length > 0
                             ? connectorTags.map((connector) => {
                                   return {
-                                      const: connector.id,
+                                      const: {
+                                          id: connector.id,
+                                          iconPath:
+                                              connector.connectors.open_graph[
+                                                  'en-US'
+                                              ].image,
```

Comment:
Still need to code the solution for this where we fetch the locale from the user properly.